.github/workflows/contributors.yml

```
@@ -36,6 +36,10 @@ jobs:
           SPONSOR_INFO: "true"
           LINK_TO_PROFILE: "true"
 
+      - name: Filter users
+        run: |
+          grep -v "author=mdn-bot" ./contributors.md > ./temp && mv ./temp ./contributors.md
```

Here's an even more elegant solution:

```suggestion
          sed -i '' '/author=mdn-bot/d' ./contributors.md
```

For multiple authors, could be (didn't test):

```
sed -i '' '/author=\(mdn-bot\|another-bot\|third-bot\)/d' ./contributors.md
```

---

I think I prefer the grep version because it's cross-platform and I can test it better locally, there's differences with macOS sed that are trickier to verify for this. For instance, this doesn't work locally:

```bash
sed -i '' '/author=\(mdn-bot\|bsmth\)/d' ./contributors.md
```